lib/matplotlib/ticker.py

```diff
@@ -548,33 +548,53 @@ def set_locs(self, locs):
             vmin, vmax = self.axis.get_view_interval()
             d = abs(vmax - vmin)
             if self._useOffset:
-                self._set_offset(d)
+                self._compute_offset()
             self._set_orderOfMagnitude(d)
             self._set_format(vmin, vmax)
 
-    def _set_offset(self, range):
-        # offset of 20,001 is 20,000, for example
+    def _compute_offset(self):
```

Comment:
Contra my reluctance to change the public API at all, changing anything with a leading `_` is fair game.